specification/schemas/Action.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "required": [
+    "action_type",
+    "values"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "action_type": {
+      "type": "string",
+      "example": "create"
```

Comment:
I think we should make this an `enum` with `create-resource` and `update-resource`, to make it clear that these are the 2 actions we (initially) support.